### ISSUE TITLE
Option to edit title, description and time-slots

### ISSUE THREAD

**Issue by muhammedShamal** [RocketMeet/RocketMeet-client#132]
After creating a poll, if the user wants to change the title, description and/or the time-slots, there is no way except creating a new poll. So it would be nice to have an option to edit the poll.

**Comment by AnandBaburajan** [comment]
Indeed, this would be quite helpful. Thanks for opening the issue! :D
